src/Login/hooks/usePostLoginToken.ts

```diff
@@ -0,0 +1,38 @@
+import { AxiosError } from 'axios';
+import { useMutation } from 'react-query';
+import { useNavigate } from 'react-router-dom';
+
+import { postLoginToken } from '../api/postLoginToken';
+
+const usePostLoginToken = () => {
+  const navigate = useNavigate();
+
+  const mutation = useMutation({
+    mutationFn: async (token: string) => {
+      return await postLoginToken(token);
+    },
+    onSuccess: (data) => {
+      const { tokenDto, nickname } = data;
+
+      if (nickname === null || nickname === '') {
```

Comment:
ASK: 요거 이렇게 바꿔보면 더 가독성이 높아질 것 같은데 어떻게 생각하시나용 !
```suggestion
      if (nickname.length === 0) {
```

Reply:
생각해보니까 빈 문자열을 넣어둔 이유가 서버에서 제 계정을 데이터베이스에서 삭제했을 때 null이 아닌 빈 string으로 초기화되어서 이를 걸러내기 위해 빈 string인지 체크하는 조건도 넣어뒀었어요..! 따라서 초기 사용자는 해당 데이터 필드가 null일 때이니까 조건을 null인지 체크하는 것만 남겨두도록 수정했습니다!!